downloads.html

```diff
@@ -76,7 +76,7 @@ <h2>Android</h2>
 
         <p>
           Alternatively,
-          <a href="https://github.com/minetest/minetest/releases/download/5.8.0/minetest-5.8.0-armeabi-v7a.apk">download the APK</a>.
+          <a href="https://github.com/minetest/minetest/releases/download/5.9.0/minetest-5.9.0-armeabi-v7a.apk">download the APK</a>.
```

Comment:
btw these days we could link the arm64 apk here too

Reply:
what about x86 and x86_64?

Reply:
yea I'd skip them